pkg/controller/node/controller_utils.go

```diff
@@ -320,3 +326,11 @@ func terminatePods(kubeClient clientset.Interface, recorder record.EventRecorder
 	}
 	return complete, nextAttempt, nil
 }
+
+func NormalLimiterFunc(nodeNum int, defaultQPS float32) float32 {
+	return defaultQPS
+}
+
+func ReducedLimiterFunc(nodeNum int, defaultQPS float32) float32 {
```

Comment:
Why do we pass number of nodes here?
If there is not a plan to to use it, we should remove it.